src/locales/en.json

```diff
@@ -1,7 +1,7 @@
 {
+  "daily_usage_comparison": "Daily usage comparison ({{units}})",
```

Comment:
Thinking this should be moved under the "chart" section

Reply:
Agreed and done